<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>469</width>
    <height>284</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestione Utenti</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(159, 197, 248)</string>
  </property>
  <widget class="QTableWidget" name="twUtenti">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>302</width>
     <height>281</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(245, 243, 201)</string>
   </property>
   <column>
    <property name="text">
     <string>Username</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Password</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Ruolo</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="btModificaUtente">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>40</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(245, 243, 201);</string>
   </property>
   <property name="text">
    <string>Modifica Utente</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
